docs/iov-name-service/validator/01-setup.md

```diff
@@ -212,4 +207,4 @@ IMAGE_TM_OPTS="\
 
 ## Light-up the validator
 
-Once your sentry nodes and validator are sync'ed then the final step to becoming a validator is to submit your validator's pub_key to IOV.  **On your validator node**, execute `curl --silent --fail http://localhost:16657/status | jq -r .result.validator_info.pub_key.value` and reply with the resulting 44 character pub_key to the ticket that was issued to you when you applied for the validator program.  (There's no `create-validator` command like in Cosmos; validators are added via governance, which is just IOV on the testnet, for the moment.)
+Once your sentry nodes and validator are sync'ed then the final step to becoming a validator is to submit your validator's pub_key to IOV.  **On your validator node**, execute `curl --silent --fail http://localhost:16657/status | jq -r .result.validator_info.pub_key.value` and reply with the resulting 44 character pub_key to the ticket that was issued to you when you applied for [the validator program here](https://support.iov.one/hc/en-us/requests/new?ticket_form_id=360000417771).  (There's no `create-validator` command like in Cosmos; validators are added via governance, which is just IOV on the testnet, for the moment.)
```

Comment:
Also maybe `[the validator program here](https://support.iov.one/hc/en-us/requests/new?ticket_form_id=360000417771)` -> `[the validator program here](https://support.iov.one/hc/en-us/requests/new?ticket_form_id=360000417771 "Validator Program")`